.github/workflows/test.yml

```diff
@@ -55,11 +55,9 @@ jobs:
       - name: SonarQube analysis
         run: gradle sonarqube -Dsonar.projectKey=$SONAR_PROJECT_KEY -Dsonar.host.url=$SONAR_HOST_URL
```

Comment:
With the SonarQube project name change can we drop SONAR_PROJECT_KEY here and remove it from the env block at the top of the file?